build-tools/automation/build.groovy

```diff
@@ -188,9 +188,9 @@ timestamps {
         stageWithTimeout('publish test error logs to Azure', 10, 'MINUTES', '', false) {  // Typically takes less than a minute
             echo "packaging test error logs"
 
-            sh "make -C ${XADir} -k package-test-errors"
+            sh "make -C ${XADir} -k package-test-results"
```

Comment:
I believe you need to add a CONFIGURATION setting here (as follows); otherwise, I found based on a test build that the package produced always carries `-Debug` configuration even for release builds.
```sh "make -C ${XADir} -k package-test-results" CONFIGURATION=${env.BuildFlavor}```

Reply:
Resolved by the following commits:
https://github.com/xamarin/xamarin-android/commit/34b122c652a979353d18fd64ad9faed07e8e4958
https://github.com/xamarin/xamarin-android/commit/cd326c08779c31d60c4d0bde9552f5be248c39eb